### DISCUSSION TITLE
'Node' object has no attribute 'get_data_type_as_variant_type'

### DISCUSSION THREAD

**Discussion by jylilaht** [FreeOpcUa/opcua-asyncio#495] [Q&A] [answered]
`myvarianttype = await myvarnode.get_data_type_as_variant_type()
AttributeError: 'Node' object has no attribute 'get_data_type_as_variant_type'`

This error appeared after updating from Python 3.6.8  to 3.9.2

Could someone please help, what has changed?

This was working all right before.

**Comment by AndreasHeine** [comment, marked as answer]
https://github.com/FreeOpcUa/opcua-asyncio/blob/4eb0edb5ab28fd267f5b6c23db08831a58eb3d4a/asyncua/common/node.py#L102

**Reply by jylilaht** [reply]
Okay thanks, this solved my problem.

Had to update get_data_type and get_data_variant_type calls.
